src/selectors/layout.js

```diff
@@ -32,35 +38,48 @@ export const getGraphInput = createSelector(
 /**
  * Calculate the displayed width of a sidebar
  */
-export const getSidebarWidth = (visible, width, { breakpoint, open, closed }) =>
-  visible && width > breakpoint ? open : closed;
+export const getSidebarWidth = (visible, { open, closed }) =>
+  visible ? open : closed;
 
 /**
  * Convert the DOMRect into an Object, mutate some of the properties,
  * and add some useful new ones
  */
 export const getChartSize = createSelector(
-  [getVisibleSidebar, getVisibleMetaSidebar, state => state.chartSize],
-  (visibleSidebar, visibleMetaSidebar, chartSize) => {
+  [
+    getVisibleSidebar,
+    getVisibleMetaSidebar,
+    getVisibleCode,
+    state => state.chartSize
```

Comment:
I didn't notice this before but this should really be `getChartSize` and defined above.